### PR TITLE
test: created tests for src/components/AddOn/core/AddOnStore/AddOnSto…

### LINKED ISSUE #1038: Code Coverage: Create tests for src/components/AddOn/core/AddOnStore/AddOnStore.tsx
We will need unittests done for all methods, classes and functions found in this file.

Any widgets, components, modals referenced in this file must also have unittests done

### PR Acceptance Criteria

- When complete this file must show at least **100%** coverage when merged into the code base. 
- [The current code coverage for the file can be found here](https://app.codecov.io/gh/PalisadoesFoundation/talawa-admin?search=&displayType=list)
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide
- Please coordinate **issue assignment** and **PR reviews** with the contributors listed in this issue https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by ALOK9442**
Fixes #1038 

**Did you add tests for your changes?**
Yes

**Snapshots/Videos:**
Not required

**If relevant, did you update the documentation?**

No documentation update required

**Summary**

I added unit tests for  src/components/AddOn/core/AddOnStore/AddOnStore.tsx

**Does this PR introduce a breaking change?**
No

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes, I have read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by ALOK9442** [comment]
@palisadoes 
I am new to this repo, please look into this whether any modifications are required. Your insights will be appreciated.

File: src/components/AddOn/core/AddOnStore/AddOnStore.test.tsx [hunk 1]
```diff
@@ -30,31 +23,301 @@ const httpLink = new HttpLink({
...
+
+    waitFor(() => {
+      expect(
+        screen.getByText('Search results for Sample Plugin')
+      ).toBeInTheDocument();
+      expect(screen.getByText('Sample Plugin 1')).toBeInTheDocument();
+    });
+  });
+
+  test('installs a plugin when "Install" button is clicked in AddOnEntry', async () => {
```
Comment (beingnoble03) [inline src/components/AddOn/core/AddOnStore/AddOnStore.test.tsx:304]: This test needs an expect statement.

**Review by beingnoble03 — CHANGES_REQUESTED** [review]
@ALOK9442 there are many tests in this PR, which need `expect` statement. Please look into it.

**Comment by ALOK9442** [comment]
I have added the expect statements, please look into this whether any modifications are required. Your insights will be appreciated.

**Comment by palisadoes** [comment]
1. This needs to be closed. You are merging against `main` versus `develop`
2. Do all your edits against the latest upstream for `develop` and propose merging your PR's branch against `develop`
3. Closing

**Comment by ALOK9442** [comment]
I tried to rebase my main with upstream/develop but facing this issue while fetching upstream
image

**Pull request closed without merging**